source/Halibut.Tests/Timeouts/SendingAndReceivingRequestMessagesTimeoutsFixture.cs

```diff
@@ -10,13 +10,57 @@
 using Halibut.TestUtils.Contracts;
 using NUnit.Framework;
 using System.Runtime.InteropServices;
+using Halibut.Tests.Support.PortForwarding;
 using Halibut.Tests.Support.TestCases;
 using Halibut.Tests.TestServices.Async;
 
 namespace Halibut.Tests.Timeouts
 {
     public class SendingAndReceivingRequestMessagesTimeoutsFixture : BaseTest
     {
+        [Test]
+        [LatestClientAndLatestServiceTestCases(testNetworkConditions: false, testWebSocket: false, testAsyncServicesAsWell: true, testSyncService : false, testSyncClients: false, testAsyncClients: true)]
+        public async Task HalibutTimeoutsAndLimits_ApplyInAsync(ClientAndServiceTestCase clientAndServiceTestCase)
```

Comment:
This is testing 1 specific timeout, can the test name be changed so it is clear it's not covering all tiemouts

Reply:
Yep